### DISCUSSION TITLE
count() error?

### DISCUSSION THREAD

**Discussion by OpenCVPytho** [prometheus/prometheus#13797] [Q&A]
### What did you do?

I want to count the point which was bigger than 10:
count( myInstant{} > 10)

### What did you expect to see?

the number of point which was bigger than 10

### What did you see instead? Under which circumstances?

I always got 1
Is this a bug or wrong way to research?

**Comment by machine424** [comment]
We'll need more detail. What are the values of `myInstant`?
I seems to be working as expected here https://prometheus.demo.do.prometheus.io/graph?g0.expr=prometheus_target_interval_length_seconds&g0.tab=0&g0.display_mode=lines&g0.show_exemplars=0&g0.range_input=12h&g1.expr=count(prometheus_target_interval_length_seconds%7B%7D%20%3E%2015)&g1.tab=0&g1.display_mode=lines&g1.show_exemplars=0&g1.range_input=1h

**Comment by beorn7** [comment]
Yeah, this is almost certainly working as intended.

Maybe you need `count_over_time`?

In any case, converting to a discussion as I strongly assume this is not a bug but about how to use `count` correctly.
